Python/wrfrun/unCMIP6.py

```diff
@@ -521,16 +587,16 @@ def write_wrfinterm(self):
             out_dic['UNIT'] = itm['units']
             if (varname=='tos' and itm['units']=='degC'):
                 out_dic['UNIT'] = 'K'
-            # NOTE: This is because in MPI-ESM1-2-HR data tos is in degress C and WRF expects degrees K.
+            # NOTE: This is because in some model data tos is in degress C and WRF expects degrees K.
```

Comment:
This is good, because it is independent of specific models.

Reply:
Great.